src/components/Header.vue

```diff
@@ -35,50 +35,51 @@ export default {
           icon: 'supervisor_account',
           title: 'VIEW MEETUPS',
           route: '/meetups',
+          registered: true,
+          guest: true,
```

Comment:
I suggest to not to use extra properties but just split data into three separate arrays (common and auth) and concatenate them for registered users